CONTRIBUTING.md

```diff
@@ -208,6 +208,29 @@ Work has been proposed to add a separate installer for AUR packages [ros-infrast
 
 For pip installers they are expected to be in the main PyPI index https://pypi.org/.
 
+### pkg-config rules
+
+For packages that contain a [pkg-config file](https://linux.die.net/man/1/pkg-config) (\*.pc) the pkg-config name must be used as rosdep key. This pkg-config name is set by the developers of a library or component and will be the same across distributions. Commonly, the very same key will be used in CMake's [`FindPkgConfig`](https://cmake.org/cmake/help/latest/module/FindPkgConfig.html) to query this dependency.
```

Comment:
Although this file isn't perfect, it does generally use semantic line endings (one sentence per line) to make the diff review process easier and less prone to style/formatting changes. Could you please update the added sections to conform to one sentence per line.

```suggestion
For packages that contain a [pkg-config file](https://linux.die.net/man/1/pkg-config) (\*.pc) the pkg-config name must be used as rosdep key.
This pkg-config name is set by the developers of a library or component and will be the same across distributions.
Commonly, the very same key will be used in CMake's [`FindPkgConfig`](https://cmake.org/cmake/help/latest/module/FindPkgConfig.html) to query this dependency.
```